frontend/.storybook/preview.js

```diff
@@ -0,0 +1,39 @@
+import { RouterContext } from "next/dist/shared/lib/router-context"
+// import { FormProvider, useForm } from "react-hook-form"
+// import { AuthProvider } from "../helpers"
+// import { addDecorator } from "@storybook/react"
+// import { initializeWorker, mswDecorator } from "msw-storybook-addon"
+
+// initializeWorker()
+// addDecorator(mswDecorator)
+
+// don't run in production or during build
```

Comment:
Not sure how storybook msw addon works but we should be able to use the useMockServiceWorker hook to set it up, as we do in app.tsx. you would call the hook in the render fn of the global decorator. It uses the mock if the env is development, not sure what storybook sets

https://github.com/codeforboston/police-data-trust/blob/main/frontend/helpers/api/mocks/browser.setup.ts

Reply:
the addon is similar and looks fun, only I didn't wire it up because there isn't really anywhere to put it - nothing takes mock data yet. but as soon as it does!